video_xblock/static/js/studio_edit.js

```diff
@@ -302,36 +302,50 @@ function StudioEditableXBlock(runtime, element) {
 
     };
 
+    var showUploadStatus = function($element, filename){
+        $element.find('.status-upload').text('File ' + '"' + filename + '"' + ' uploaded successfully').show();
+        setTimeout(function(){
+            $('.status-upload', $element).hide()
+        }, 5000);
+    };
+
     $fileUploader.on('change', function(event) {
         var fieldName = $(event.currentTarget).attr('data-change-field-name');
         var lang = $(event.currentTarget).attr('data-lang-code');
         var label = $(event.currentTarget).attr('data-lang-label');
         var currentLiIndex = $(event.currentTarget).attr('data-li-index');
         var currentLiTag = $('.language-transcript-selector').children()[parseInt(currentLiIndex)];
         $('.upload-setting', element).addClass('is-disabled');
-        $('.file-uploader-form', element).ajaxSubmit({
-            success: function(response, statusText, xhr, form) {
-                if(fieldName == "handout"){
-                    $('input[data-field-name=' + fieldName + ']').val(response['asset']['id']).change();
-                } else {
+        if($fileUploader.val()){
```

Comment:
Please invert this if-clause to short-circuit in case if `.val()` is empty.
So that normal flow is outside if block.